notes/Module-2/2.2.4-Integration_tests.md

```diff
@@ -0,0 +1,12 @@
+# Coach Notes
+
+## Coach comments
+Please add comments below which will help future coaches delivering this topic or suggestions for improvement of the content.
+
+|**Coach name**|**Delivery date**|**Comments**|
+|--------------|-----------------|------------|
+|Mandy|13/05/21|My understanding of static tests is pre-compilation time i.e. when you use tools like Sonar or manual code reviews to run static analysis of code. Could we rephrase to talk about unit versus integration testing? |
```

Comment:
This distinction between Static and Dymamic tests is drawn from the BCS software methodologies syllabus and will come up again in Mod 3. Static and Dymamic tests are different from Unit and Integration tests. Each of these terms will need explaining separately.